### DISCUSSION TITLE
Quarkus RestEasy Reactive + multipart/mixed

### DISCUSSION THREAD

**Discussion by sahilgupta-gti** [quarkusio/quarkus#27557] [Q&A]
I could not find documentation for Quarkus RestEasy Reactive for using Multipart/mixed.
Is it possible to use multipart/mixed with Quarkus RestEasy Reactive, if yes please help me on this.

Since same can be done with Quarkus RestEasy with multipart provider but same is not working with RestEasy Reactive.

[a comment by an automated account is not shown]

**Comment by geoand** [comment]
What exactly are you trying to do? Can you show us a sample?

**Reply by sahilgupta-gti** [reply]
Hi @geoand,

We are migrating are existing application on Quarkus with "quarkus-resteasy-reactive-jackson" extension and in existing application we are accepting the data with Content-Type="multipart/mixed".

I checked the Quarkus documentation, Content-Type = mutlipart/mixed is supported only in "quarkus-resteasy-jackson" with extension "quarkus-resteasy-multipart" but not with "quarkus-resteasy-reactive-jackson".

Is there any chance to have same support of multipart/* as we have in classic Resteasy?

Following is the sample which we want to support in our application. I know in reactive variant Quarkus by default support multipart/form-data, but requiring multipart/mixed like it was in classic Resteasy.


```
@POST
@Consumes("multipart/mixed")
@Path("/multipart")
public Response multipart(SomePojo somePojo)  {
//Do some task
}

```

**Reply by geoand** [reply]
Can you please open an issue asking for such support? It would be very helpful if you included a sample project that works now with RESTEasy Classic and a sample client request.
